package.json

```diff
@@ -91,6 +91,9 @@
     "ts-node": "^10.2.1",
     "typescript": "^4.4.3"
   },
+  "publishConfig": {
+    "access": "public"
```

Comment:
I don't fully understand why this is needed but It's because you know, not familiar with it at all. Would you mind explaining quickly to me, please 🙏 ?

Reply:
Hah it's only for release on npm, which we don't do. I'll have to remove that and let you know. But the general idea is to be able to publish the public version on npm the scoped packages (in our case `@asyncapi/*`), which by default is `restricted`. More info https://docs.npmjs.com/cli/v8/using-npm/config#access

Reply:
Ok removed, @smoya Could you check again? Thanks! :)